commands/connection_pooling.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const host = require('../lib/host')
+
+function * run (context, heroku) {
+  const util = require('../lib/util')
+  const fetcher = require('../lib/fetcher')(heroku)
+  const {app, args, flags} = context
+
+  let db = yield fetcher.addon(app, args.database)
+  let addon = yield heroku.get(`/addons/${encodeURIComponent(db.name)}`)
+  let credential = `${flags.credential || 'default'}`
+
+  if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
+
+  let attachment = yield cli.action(
+    `Enabling Connection Pooling${credential === 'default' ? '' : ' for credential ' + cli.color.addon(credential)} on ${cli.color.addon(addon.name)} to ${cli.color.app(app)}`,
+    heroku.post(`/client/v11/databases/${db.name}/connection-pooling`, {
```

Comment:
Why not `encodeURIComponent(db.name)` here? I mean, I don't think it really matters--I don't believe add-on names support any URL-unsafe characters--but we should be consistent.